pkg/proxy/iptables/proxier_test.go

```diff
@@ -458,7 +458,7 @@ func parseIPTablesData(ruleData string) (map[string][]string, error) {
 	return tables, nil
 }
 
-func Test_parseIPTablesData(t *testing.T) {
+func TestParseIPTablesData(t *testing.T) {
```

Comment:
This is a not-uncommon convention for testing functions with initial-lowercase names:

    danw@p1:kubernetes> cat $(find pkg -name \*_test.go) | grep -c '^func Test_[a-z]'
    65

Reply:
I see.  It's still ... ugh  I wish the norm was ALWAYS Test_.

Are we going to argue this?  I can partially revert it if you feel strongly :)